source/feature-documentation/core/pipelines/hdr.rst

```diff
@@ -0,0 +1,22 @@
+HDR Pipeline
+============
+
```

Comment:
HDR Panorama Workflow

    Fusion of Multi-bracketing LDR images into HDR
    Alignment of panorama images
    Support for fisheye optics
    Automatically estimate fisheye circle or manually edit it
    Take advantage of motorized-head file